deepspeed/elasticity/elastic_agent.py

```diff
@@ -6,7 +6,7 @@
 from torch.distributed.elastic.agent.server.local_elastic_agent import LocalElasticAgent
 from typing import Any, Dict, Optional, Tuple
 from datetime import datetime
-from torch.distributed.elastic.agent.server.api import _get_socket_with_port
+from torch.distributed.elastic.utils.distributed import get_free_port as _get_free_port
```

Comment:
Just use `get_free_port` at the calling site instead of importing as old private name?